website/orders/admin.py

```diff
@@ -79,13 +76,6 @@ class Media:
         """Necessary to use AutocompleteFilter."""
 
 
-class OrderInline(admin.TabularInline):
-    """Inline form for orders."""
-
-    model = Order
```

Comment:
Yes, even when optimized a lot, there are (many) queries per order. And rendering all of those fields to html (even with non-dropdown widgets for user) is quite slow too. And yes it's really inconvenient, especially since saving a model with the inlines there overrides all changes from the vue admin that happened in the mean time.

Reply:
+1